### ISSUE TITLE
Comparative and Score are coming out the same

### ISSUE THREAD

**Issue by sseetal** [thisandagain/sentiment#111]
I have version 2.1.0 on both my computer and on a server. On my computer it works, on my server, I get the following

{ score: 3,
  comparative: 3,
  tokens: [ 'happy' ],
  words: [ 'happy' ],
  positive: [ 'happy' ],
  negative: [] }

{ score: -2,
  comparative: -2,
  tokens: [ 'sad' ],
  words: [ 'sad' ],
  positive: [],
  negative: [ 'sad' ] }

{ score: 0,
  comparative: 0,
  tokens: [ 'ok' ],
  words: [],
  positive: [],
  negative: [] }

Any thoughts?

**Comment by thisandagain** [comment]
@sseetal Can you provide a little more info?

- What versions of node are you running on the server and your computer?
- What are the input strings that are producing these results?

**Comment by sseetal** [comment]
Thanks for getting back to me. 

1. We're using nvm everywhere. node 4.1.1 locally (MAC), v6.9.4 on Server (Ubuntu)
2. The input streams are in the results I posted above (happy, sad, ok) to try and get the three desired results (positive, negative and neutral results)
3. I tried getting the local results (the server results are above). I can now further understand the problem. It's because they are one word. 

INPUT sentiment

For input "happy" (both locally and on server)

{ score: 3,
  comparative: 3,
  tokens: [ 'happy' ],
  words: [ 'happy' ],
  positive: [ 'happy' ],
  negative: [] }

For Input "I am happy today"

{ score: 3,
  comparative: 0.75,
  tokens: [ 'i', 'am', 'happy', 'today' ],
  words: [ 'happy' ],
  positive: [ 'happy' ],
  negative: [] }

**Comment by sseetal** [comment]
Just to add more context. We're using this to analyze comments left by people in a survey. The whole point is to have a score and an overall score. Ideally a score out of 100. But can't seem to figure out how to calculate a comparable score.

**Comment by thisandagain** [comment]
[This issue](https://github.com/thisandagain/sentiment/issues/107) includes a thread that covers some of the areas you are asking about including how "comparative scores" work and how to scale / offset them. 

TL;DR: Tokens are scored between -5 (highly negative) and 5 (highly positive) with 0 being "neutral". Comparative scores are simply SUM(tokens) / number of tokens. Details can be found in the AFINN whitepaper:
http://www2.imm.dtu.dk/pubdb/views/publication_details.php?id=6010
